CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## 1.1.0
```

Comment:
We already created [release](https://github.com/Microsoft/powerbi-visuals-gantt/releases/tag/1.1.0) for this version on GitHub.
Please increase version of the visual.